data/SARSCoV2_Vitro_Touret/meta.yaml

```diff
@@ -0,0 +1,38 @@
+name: SARSCoV2_Vitro_Touret
+description: An in-vitro screen of the Prestwick chemical library composed of 1,480
+  approved drugs in an infected cell-based assay. From MIT AiCures.
+targets:
+- id: activity_SARSCoV2
+  description: whether it active against SARSCoV2 (1) or not (0).
+  units: activity
+  type: categorical
+  names:
+  - Corona activity
+  - activity
+  - activity against SARSCoV2
+  - COVID19
+  - Coronavirus disease
+  - Coronavirus
```

Comment:
It would be great if all contained activity vs. corona in some from. 
They will be used to prompt the model and we cannot expect 0/1 to be a completion for "covid" or "coronavirus"